test/integration/shared_vpc/controls/shared_vpc.rb

```diff
@@ -0,0 +1,64 @@
+# Copyright 2018 Google LLC
```

Comment:
These tests appear to be behaving as expected (once the host integration tests are commented out):

```
  ✔  forseti-service-project: test forseti project
     ✔  Compute Project Info thebo-forseti-svc-d296 xpn_project_status should eq "UNSPECIFIED_XPN_PROJECT_STATUS"
     ✔  Compute Project Info thebo-forseti-svc-d296 name should eq "thebo-forseti-svc-d296"
  ✔  forseti-shared-project: test shared project
     ✔  Compute Project Info thebo-forseti-927c xpn_project_status should eq "HOST"
     ✔  Compute Project Info thebo-forseti-927c name should eq "thebo-forseti-927c"
  ✔  forseti-server: test forseti server
     ✔  Instance forseti-server-vm-2546f87b should exist
     ✔  Instance forseti-server-vm-2546f87b has_disks_encrypted_with_csek? should equal false
     ✔  Instance forseti-server-vm-2546f87b status should eq "RUNNING"
     ✔  Instance forseti-server-vm-2546f87b disk_count should eq 1
  ✔  forseti-client: test forset client
     ✔  Instance forseti-client-vm-2546f87b should exist
     ✔  Instance forseti-client-vm-2546f87b has_disks_encrypted_with_csek? should equal false
     ✔  Instance forseti-client-vm-2546f87b status should eq "RUNNING"
     ✔  Instance forseti-client-vm-2546f87b disk_count should eq 1
```